pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -557,6 +557,23 @@ public static class Minion {
     public static final String CONFIG_OF_MINION_QUERY_REWRITER_CLASS_NAMES = "pinot.minion.query.rewriter.class.names";
   }
 
+  public static class ControllerJob {
+    /**
+     * Controller job ZK props
+     */
+    public static final String CONTROLLER_JOB_TYPE = "jobType";
```

Comment:
(minor) We may also simplify the key name because it is under `ControllerJob` class
```suggestion
    public static final String JOB_TYPE = "jobType";
```

Reply:
Ack